ceph.spec.in

```diff
@@ -804,7 +804,6 @@ rm -rf $RPM_BUILD_ROOT
 
 
 %post
```

Comment:
@ktdreyer I guess we could do that. I was thinking that the `ceph.target` systemd service belongs to the `ceph` package (i.e. to the unnamed `%post` etc. sections).